db/seeds.rb

```diff
@@ -5,3 +5,5 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+User.create(id: 0, cdx_user_id: 'unknown_user')
```

Comment:
should we have a note in the readme about this? (not obvious to be why we need it / how it is used)